arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -545,6 +545,104 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/powerOff": {
+      "post": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "Redis_PowerOff",
+        "x-ms-examples": {
+          "RedisCachePowerOff": { "$ref": "../examples/RedisCachePowerOff.json" }
+        },
+        "x-ms-long-running-operation": true,
+        "description": "PowerOff an existing redis cache.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Redis cache."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "PowerOff operation successfully enqueued; follow the Location header to poll for final outcome.",
+            "schema": {
+              "$ref": "#/definitions/RedisPowerOffResponse"
+            }
+          },
+          "200": {
+            "description": "PowerOff operation succeeded.",
+            "schema": {
+              "$ref": "#/definitions/RedisPowerOffResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/start": {
+      "post": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "Redis_Start",
+        "x-ms-examples": {
+          "RedisCacheStart": { "$ref": "../examples/RedisCacheStart.json" }
+        },
+        "x-ms-long-running-operation": true,
+        "description": "Start an existing redis cache.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Redis cache."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Start operation successfully enqueued; follow the Location header to poll for final outcome.",
+            "schema": {
+              "$ref": "#/definitions/RedisStartResponse"
```

Comment:
OK, so we should remove the 'schema' line then - if that response is the response eventually returned by the long running operation, would it make sense to list that schema with a hypothetical "200" success response code?